src/frontend/src/pages/TeamsPage/Teams.tsx

```diff
@@ -6,12 +6,12 @@
 import { Route, Switch } from 'react-router-dom';
 import { routes } from '../../utils/routes';
 import TeamsPage from './TeamsPage';
-import TeamSpecificPage from './TeamSpecificPage';
+import TeamDetailPage from './TeamSpecificPage';
 
 const Teams: React.FC = () => {
   return (
     <Switch>
-      <Route path={routes.TEAMS_BY_ID} component={TeamSpecificPage} />
+      <Route path={routes.TEAMS_BY_ID} component={TeamDetailPage} />
```

Comment:
what's goin on here?

Reply:
Oh I was messing with some naming but figured it wasn't worth it - mustve forgotten to change that line back